### ISSUE TITLE
32/yf cannot enable saturation splines

### ISSUE THREAD

**Issue by ianhbell** [usnistgov/REFPROP-issues#340]
For some reason, perhaps related to #334, it seems to be impossible to programmatically use the saturation spline for the mixture R32+R1234yf, but it works in the GUI.  In code, this gives the wrong density:
``` python
import os
from ctREFPROP.ctREFPROP import REFPROPFunctionLibrary
os.environ['RPPREFIX'] = 'D:/Program Files (x86)/REFPROPalpha'
root = os.environ['RPPREFIX']
RP = REFPROPFunctionLibrary(root)
print(RP.RPVersion())

iFlag = 0
iSpln = 1
z = [0.8122,0.1878]
T, p = 263.15, 537.4*1e3
flds = 'R32*R1234YF'
RP.SETFLUIDSdll(flds)
if iSpln == 1:
    print(RP.SATSPLNdll(z))
r = RP.REFPROPdll('', 'TP', 'D', RP.MOLAR_BASE_SI, iFlag, iSpln, T, p, z)
print(r.Output[0])
```
yielding
```
10.0085
SATSPLNdlloutput(ierr=0, herr='')
1052.8468368161289
```
In the GUI, when I do not enable the saturation splines I get the top panel (wrong result), and I get the bottom panel (correct result) when I enable saturation splines:
![image](https://user-images.githubusercontent.com/11966765/95387188-5d79bd80-08be-11eb-9a04-4573855d98d7.png)

**Comment by ianhbell** [comment]
On digging deeper, I believe the no splines solution is the "right" one in this case, although it doesn't agree with the data.  The value with the splines on is outside the saturated liquid and vapor values at the specified mixture composition and temperature.

The original issue stands. There is something funny with R32+R1234yf

**Comment by EricLemmon** [comment]
Can you try this again with the current code?  I tried it here (just with the GUI) and got the same thing with and without the splines on.

**Comment by ianhbell** [comment]
Which value did you get?

**Comment by EricLemmon** [comment]
To make sure that I really had the splines off in one case and on in the other one, I started two separate versions of the GUI, both running simultaneously.  I first turned off the splines in one and ran the calculation, and then simply ran the same calculation in the other one (where the splines are on by default).  Both produced the same thing, and upon opening the Substance/Fluid Information screen in each I can see that one has the splines on and one does not.  A picture is shown below, with both reporting a density of 1.05285 mol/l.  

Then to verify that there was no connection between two Refprops running, both using the same DLL and same fluid files, I saved the options in the one that did not have the splines on, closed both programs, reopened Refprop (just one time) and calculated the values directly.  Then I closed and restarted the program, turned on the splines, and calculated the values.  There was no change in either situation.  So there does not seem to be any memory issues or cross connecting between the DLLs as well in these mini-tests.

Identical calculated values can also be obtained with the process you used by calculating the value first with the splines on, going to the options/preferences and turning off the splines, and then recalculating the points.  Both ways produce the same density.


![image](https://user-images.githubusercontent.com/42392984/112767439-0e425900-8fd4-11eb-82d0-5fb71393206e.png)
